### ISSUE TITLE
Send email with Workflow using Meetings module breaks parent_id

### ISSUE THREAD

**Issue by sanchezfauste** [salesagility/SuiteCRM#5971]
#### Issue

If you create a Workflow using as main module Meetings and you send an email to related record of Meeting, the `parent_id` field of Meeting is replaced by the id of the Email record.

#### Expected Behavior

If you send an Email to the related record of a Meeting, `parent_id` field of the Meeting must remain with the actual value.

#### Actual Behavior

The `parent_id` of the Meeting is replaced by the id of the sended email.

#### Possible Fix

Just observe the following lines:
https://github.com/salesagility/SuiteCRM/blob/bb0f2ddb22921d7301abb82a3735d015c8422327/modules/Emails/vardefs.php#L698-L706

I will submit a PR later.

#### Steps to Reproduce

1. Create the following workflow.
![1](https://user-images.githubusercontent.com/10921800/40839220-db49315e-65a1-11e8-91ef-c76307f53509.png)
![2](https://user-images.githubusercontent.com/10921800/40839223-dd0fee56-65a1-11e8-9922-c73dbc978996.png)
(create an email template with whatever text)

2. Create a new Meeting related with a Lead that has a valid email.
3. Wait until the mail is sent.
4. Open the afected Meeting and observe that Lead "has disappeared".
5. Now it's time to do some SQL querys:
```
SELECT parent_type, parent_id FROM meetings WHERE id = '<meeting_id>'
```
6. See that `parent_type` is correct but `parent_id` of `meetings` table corresponds to an id of `emails` table.

#### Context

This type of workflows are breaking the related parent.

#### Your Environment

* SuiteCRM Version used: 7.8.19
* Browser name and version (e.g. Chrome Version 51.0.2704.63 (64-bit)): Versión 66.0.3359.181 (Build oficial) (64 bits)
* Environment name and version (e.g. MySQL, PHP 7): PHP 7.0.30-0ubuntu0.16.04.1 and MySQL 5.7.22-0ubuntu0.16.04.1
* Operating System and version (e.g Ubuntu 16.04): Ubuntu 16.04

**Comment by LuminisIndia** [comment]
@sanchezfauste 
The changes made by you in your suggested file: modules/Emails/vardefs.php are NOT at right line # numbers.  You need to please made more clear reference to the existing file withing this original repository of https://github.com/salesagility/SuiteCRM.  It seems that you have earlier edited your own vardefs.php before, making reference to line 530, 530 (meant to be deleted).  Thanks!

**Comment by sanchezfauste** [comment]
Hi @LuminisIndia,

My changes are done against `hotfix-7.8.x` branch as is explained here: https://docs.suitecrm.com/community/contributing-code/bugs/ because this bug is present on 7.8.x, 7.9.x and 7.10.x.

It's true that in the `master` branch the definition of `emails_meetings_rel` is not on line 530 and maybe a merge conflict will occur. This will be corrected when `hotfix-7.8.x` will be merged on `hotfix` branch.

If you want to test my changes in a version of SuiteCRM different from 7.8.x, you should apply them manually.
